js/ui/slider/ui.slider.js

```diff
@@ -520,10 +548,24 @@ const Slider = TrackBar.inherit({
     },
 
     _setValueOnSwipe: function(value) {
+        this._actualValue = value;
+
+        if(this.option('callValueChange') === 'onMovingComplete') {
+            SliderHandle.getInstance(this._activeHandle()).option('value', value);
+            return;
```

Comment:
it's better to use `if-else` instead of such unexpected `return` to have only one function end point and simplify future debug